standalone-examples/src/main/java/io/pravega/example/consolerw/ConsoleWriter.java

```diff
@@ -59,10 +59,9 @@
             "",
             "WRITE_EVENT {event} - write the {event} out to the Stream or the current Transaction.",
             "WRITE_EVENT_RK <<{routingKey}>> , {event} - write the {event} out to the Stream or the current Transaction using {routingKey}. Note << and >> around {routingKey}.",
-            "BEGIN [{transactionTimeout}] [, {maxExecutionTime}] [, {scaleGracePeriod}] begin a Transaction. Only one Transaction at a time is supported by the CLI.",
```

Comment:
Did you find it too inconvenient to set `transactionTimeout` and `transactionTimeoutScaleGracePeriod` on the `EventWriterConfig`?

Reply:
We don't need an explicit BEGIN transaction option since the writer is initialized early in the program and `EventWriterConfig` is passed with appropriate transaction timeout configurations.